core/src/main/java/org/kiwix/kiwixmobile/core/di/modules/ActivityModule.kt

```diff
@@ -68,4 +66,7 @@ abstract class ActivityModule {
       )
     }
   }
+  @Provides
+  @Named("activity")
+  fun provideActivity(): Activity = Activity()
```

Comment:
this should not be done. If we need an activity instance we pass it in to the ActivityComponent via the ActivityComponent.Builder